contrib/distrobox/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "distrobox"
+pkgver = "1.7.1.0"
+pkgrel = 0
+depends = ["curl"]
+pkgdesc = "Use any Linux distribution inside your terminal"
+maintainer = "Jami Kettunen <jami.kettunen@protonmail.com>"
+license = "GPL-3.0-only"
+url = "https://distrobox.it"
+source = f"https://github.com/89luca89/distrobox/archive/{pkgver}.tar.gz"
+sha256 = "71af97c7033cdd3f2679e922b52e5b6c801d697af63bbe3f9d771948c6575e90"
+
+
+def do_install(self):
+    self.do("./install", "--prefix", f"{self.chroot_destdir}/usr")
+    # conf
+    self.install_file(self.files_path / "distrobox.conf", "usr/share/distrobox")
+    # docs
+    self.install_file("docs/*.md", "usr/share/doc/distrobox", glob=True)
+
+
+@subpackage("distrobox-podman")
```

Comment:
Technically `docker-cli` also should work here but I wasn't sure how to make the backend selection

Reply:
Ended up dropping this for now since apparently rootless containers don't work with it, let me know if I should make `podman` a soft-dep still